Export/Skills/spectre.txt

```diff
@@ -713,13 +713,24 @@ local skills, mod, flag, skill = ...
 #skill SandLeaperDodgeRight Sand Leaper Dodge Right
 #mods
 
+#skill SynthesisSoulstealerProjectileLightning Lightning Projectile
+#flags spell triggerable projectile
+#mods
+
+#skill SynthesisSoulstealerLaser Lightning Laser
+#flags spell hit area triggerable
+#mods
+
+#skill SynthesisSoulstealerBolt Lightning Bolt
+#flags spell hit area triggerable
+#mods
+
 #skill MeleeCold Default Attack
 #flags attack melee projectile
 #mods
 
 #skill AtlasCrusaderJudgeBallLightning Ball Lightning
 #flags spell hit triggerable area projectile
-#mods
```

Comment:
That seems to be an editing error. The rest looks good though. Please fix it so we can accept your PR.